.github/workflows/upgrade-operator.yaml

```diff
@@ -57,7 +57,7 @@ jobs:
             echo "Exiting the workflow as there is no change to generate a PR."
             exit 1
           fi
-          git add .
+          git add -A .
```

Comment:
nit: `git add -A` is the same as `git add -A .`, so the `.` is redundant.

Reply:
Thanks for pointing it out! I also checked the latest manual and experimented with the command locally, and found that `git add .` should already add all the modified, added and deleted files: https://git-scm.com/docs/git-add#Documentation/git-add.txt-ltpathspecgt82308203